INSTALL

```diff
@@ -2,4 +2,4 @@ Building Bitcoin
 
 See doc/build-*.md for instructions on building bitcoind,
 the intended-for-services, no-graphical-interface, reference
-implementation of Bitcoin.
\ No newline at end of file
+implementation of Bitcoin.
```

Comment:
It is not a pointless change. That file is used by sysadmin to install the software. And when you are a sysadmin and "cat INSTALL.md" does not bring you on the same line, you think you missed something of the original package.